PowerEditor/src/Notepad_plus.cpp

```diff
@@ -174,6 +174,11 @@ Notepad_plus::Notepad_plus()
 		is_admin = false;
 
 	_isAdministrator = is_admin ? true : false;
+
+	// Enable new style save dialog.
+	if (ver >= WV_VISTA) {

```

Comment:
Why new style dialog is applicable only for vista and higher version?

As per [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/bb773205(v=vs.85).aspx) , minimum version of Shell dll should be 5.0 and 6.0 for BIF_USENEWUI  and BIF_NONEWFOLDERBUTTON respectively. So I think XP also should be covered.

![image](https://cloud.githubusercontent.com/assets/14791461/20808871/3d27a74c-b82a-11e6-93b6-725f7cb4a0d3.png)